### PR TITLE
D8CORE-1197 Added google form media type

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- this is dependent on https://github.com/SU-SWS/stanford_media/pull/49
- New media type for google forms

# Need Review By (Date)
- 4/15

# Urgency
- medium

# Steps to Test
1. checkout the branch in stanford_basic
1. checkout this branch
1. clear cache
1. `drush cim -y`
1. a page with a wysiwyg and add a google form.
1. validate it embeds.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: config/sync/seckit.settings.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+seckit_xss:
```
Comment (sherakama) [inline config/sync/seckit.settings.yml:1]: Snuck that in here eh?

File: config/sync/core.entity_form_display.media.google_form.default.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+uuid: 84b810f0-d451-4792-b08e-09e64af9d278
```
Comment (sherakama) [inline config/sync/core.entity_form_display.media.google_form.default.yml:1]: Why not have this config in stanford_media?

File: config/sync/seckit.settings.yml (hunk 1 above)
Comment (pookmish) [reply config/sync/seckit.settings.yml:1]: it came out during an export so i figured what the heck.

File: config/sync/core.entity_form_display.media.google_form.default.yml (hunk 2 above)
Comment (pookmish) [reply config/sync/core.entity_form_display.media.google_form.default.yml:1]: we could or we dont need to. i have no objections either way. it just comes down to the question: Should new sites that are just starting (not the ones in on this repo) to use stanford media get the media type?

Comment (sherakama) [reply config/sync/core.entity_form_display.media.google_form.default.yml:1]: That is where we are managing the other media types. I think it makes sense to me.

Comment (sherakama) [reply config/sync/core.entity_form_display.media.google_form.default.yml:1]: Please move it along

Comment (pookmish) [reply config/sync/core.entity_form_display.media.google_form.default.yml:1]: https://github.com/SU-SWS/stanford_media/pull/52

**Comment by sherakama** [comment]
Thank you.

**Pull request merged**